docs/using/output.md

```diff
@@ -63,9 +54,15 @@ By default, this will separate non-bonded interactions into several different
 `openmm.Force` objects. To combine everything into a single
 `openmm.NonbondedForce`, use the `combine_nonbonded_forces=True` argument.
 
+## Amber
 
+Under construction!
```

Comment:
We can split this out into a separate PR, but this section is no longer under construction